lib/dependencies/ImportParserPlugin.js

```diff
@@ -30,6 +30,8 @@ class ImportParserPlugin {
 
 			let chunkName = null;
 			let mode = "lazy";
+			let filter = null;
```

Comment:
Rename `filter` to `include` to align it with `include` `exclude` in `module.rules`